appbuilder/project/project-base.ts

```diff
@@ -1,29 +1,51 @@
+import {EOL} from "os";
 import Future = require("fibers/future");
-import { TARGET_FRAMEWORK_IDENTIFIERS } from "../../mobile/constants";
+import { TARGET_FRAMEWORK_IDENTIFIERS } from "../../constants";
 import * as path from "path";
-import { startPackageActivityNames } from "../../mobile/constants";
+import { startPackageActivityNames } from "../../constants";
+export abstract class ProjectBase implements Project.IProjectBase {
+	private static CONFIGURATION_FILE_SEARCH_PATTERN: RegExp = new RegExp(".*.abproject$", "i");
```

Comment:
If the user creates configuration file with name which contains invalid characters for JavaScript object property this regular expression will match it as valid configuration name. I think `^.(?:[A-z\d]+.)?abproject$` will be better.